src/build/buildNativefierApp.ts

```diff
@@ -65,9 +65,8 @@ async function copyIconsIfNecessary(
   }
 
   // windows & linux: put the icon file into the app
-  const destAppPath = path.join(appPath, 'resources/app');
   const destFileName = `icon${path.extname(options.packager.icon)}`;
-  const destIconPath = path.join(destAppPath, destFileName);
+  const destIconPath = path.join(appPath, destFileName);
```

Comment:
Also: so now, icons would no longer live in `<path_to_app>/resources/app`, but instead in `<path_to_app>`. Was it necessary for your patch? I liked it in `<path_to_app>/resources/app`

Reply:
Icons will continue to live in `<path_to_app>/resources/app`.

The current patch copies icons to the pre-stage tmp folder, `appPath`. When electron-packager runs via `const appPathArray = await electronPackager(options.packager);` everything in the pre-stage folder gets moved into `<path_to_app>/resources/app`.

Rather than copy icons directly into `<path_to_app>/resources/app`, the patch allows electron-packager to handle the icon move thus enabling the asar function, while also preserving the `<path_to_app>/resources/app/icon.ext` scheme regardless of the `--conceal` option.

Reply:
without `--conceal`

![image](https://user-images.githubusercontent.com/27786172/100639276-f6e3af80-3302-11eb-86e0-f468a9dc7f18.png)

Reply:
with `--concel`
![image](https://user-images.githubusercontent.com/27786172/100639457-30b4b600-3303-11eb-9f14-d6ee6966c503.png)

Reply:
Sorry, I should have done a better job explaining it.

Reply:
No worry, and thanks for the details/screenshots 👍